packages/design-system/src/components/Alert/Alert.tsx

```diff
@@ -218,7 +224,19 @@ export class Alert extends React.PureComponent<
       >
         {this.getIcon()}
         <div className="ds-c-alert__body">
-          {this.heading()}
+          {heading ? (
+            <div className="ds-c-alert__header ds-c-alert__heading">
+              <span className="ds-c-alert__a11y-label ds-u-visibility--screen-reader">
+                {variation ? this.a11yLabel[variation] : 'Notice'}:{' '}
+              </span>
+              {this.heading()}
+            </div>
+          ) : (
+            <span className="ds-c-alert__a11y-label ds-u-visibility--screen-reader">
+              {variation ? this.a11yLabel[variation] : 'Notice'}:{' '}
+            </span>
+          )}
+
           {children}
```

Comment:
Children will always render as a block element beneath the label. I considered adding some CSS to make the label appear inline with the children, but not knowing what children a user could pass made me reconsider this. I opted to treat the label as a backup heading instead.